File: ruby/delivery-api/delivery_api_get_type.rb
# Tip: Find more about Ruby SDKs at https://kontent.ai/learn/ruby
require 'delivery-sdk-ruby'

delivery_client = Kontent::Ai::Delivery::DeliveryClient.new project_id: '<YOUR_ENVIRONMENT_ID>'
delivery_client.type('article').execute do |response|
  type = response.type
end

File: java/delivery-api/delivery_api_get_type.java
// Tip: Find more about Java SDK at https://kontent.ai/learn/java
import kontent.ai.delivery.*;

// Initializes a DeliveryClient
DeliveryClient client = new DeliveryClient("KONTENT_AI_ENVIRONMENT_ID");

// Gets a content type
CompletionStage<ContentType> type = client.getType("article");
